vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -0,0 +1,125 @@
+# Introduction
```

Comment:
I think we need the 2nd level nav in the left bar so add this to the top:
```
---
sidebarDepth: 2
---
```

Reply:
I was wondering why the nav was incomplete, thought I was confused about the header level that was included but now it makes sense. :)